scripts/auth_test.php

```diff
@@ -106,7 +106,12 @@
     `stty echo`;
     echo PHP_EOL;
 
-    echo "Authenticate user $test_username: \n";
+    if (Config::get('auth_mechanism') == "active_directory" && strpos($test_username, "@")) {
+        $test_username = $authorizer->getUsersAMAccountName($test_username);
```

Comment:
don't update $test_username here, only store it for printing

Reply:
$test_username is used to authenticate, so it must be convert to sAMAccountName if UserPrincipalName is provided.

If I don't change it here, I think about this code to indicate to the user that it will use the sAMAaccountName (provided username) `Authenticate user dgsd (dogus@lab.local): `, but it will call 2 times getUsersAMAccountName (so 2 LDAP Search) :/

<img width="871" alt="Screen Shot 2020-03-25 at 11 23 37 am" src="https://user-images.githubusercontent.com/6872180/77526582-1fb9c800-6e8b-11ea-9e26-c3d50b5c4ae8.png">